projects/DensePose/doc/GETTING_STARTED.md

```diff
@@ -9,6 +9,11 @@ python apply_net.py show configs/densepose_rcnn_R_50_FPN_s1x.yaml densepose_rcnn
 ```
 Please see [Apply Net](TOOL_APPLY_NET.md) for more details on the tool.
 
+DensePose pre-trained models have been made available in the form of hosted APIs by third parties 
```

Comment:
We are already adding it in 2 README.md files.  Adding in GETTING_STARTED.md as well doesn't give us much value.  It might "seem" that we are trying to add "Tiyaro" branding in multiple files of this repo.  I would suggest one file modification for base detectron2 models and one file modification for densepose models